modules/setting/log.go

```diff
@@ -113,7 +112,6 @@ func generateLogConfig(sec *ini.Section, name string, defaults defaultLogOptions
 			panic(err.Error())
 		}
 
-		logConfig["colorize"] = sec.Key("COLORIZE").MustBool(runtime.GOOS != "windows")
```

Comment:
So file logs always have no colorize?

Reply:
Nope it falls through to:

https://github.com/zeripath/gitea/blob/be7956b573a3bee81ab03ff6b215700729c83d51/modules/setting/log.go#L135

So it just defaults to false